Flow.Launcher/Languages/en.xaml

```diff
@@ -93,6 +93,7 @@
     <system:String x:Key="typingStartEn">Always Start Typing in English Mode</system:String>
     <system:String x:Key="typingStartEnTooltip">Temporarily change your input method to English mode when activating Flow.</system:String>
     <system:String x:Key="autoUpdates">Auto Update</system:String>
+    <system:String x:Key="autoUpdatesTooltip">Automatically check app updates and notify if there are any updates available</system:String>
```

Comment:
This auto update for the app actually updates so the description needs a bit more specific.